src/rf/apps/workers/process.py

```diff
@@ -30,6 +30,8 @@
     'ObjectCreated:Post',
     'ObjectCreated:Copy'
 ]
+# Used when validating an image to kick off a retry attempt
+JOB_RETRY_VALIDATE = 'retry_validate'
```

Comment:
Why create a separate job for this rather than just throwing the original validation message in the queue?